infra/modules/azure_function_app/locals.tf

```diff
@@ -0,0 +1,6 @@
+locals {
+  project = "${var.prefix}-${var.env_short}-${var.location_short}"
+
+  sku_name               = var.env_short == "p" ? "P1v3" : "B1"
```

Comment:
consider that the development sku should have the same networking capabilities of the production one otherwise it might be unfeasible to connect the function with other services.

Reply:
Yesterday I created the resource with the "dev" sku and I didn't notice any issue. I'm still waiting to deploy an application on that to ensure everything run smoothly

Reply:
I realized the staging slot is not supported out of premium plans. Is it an issue for dev resources? @lucacavallaro

Reply:
As discussed, we plan to introduce a variable where combination among sku scope and slots are tighten together and exposed as enum to consumer